common/djangoapps/third_party_auth/models.py

```diff
@@ -88,7 +88,7 @@ class ProviderConfig(ConfigurationModel):
     """
     Abstract Base Class for configuring a third_party_auth provider
     """
-    KEY_FIELDS = ('slug',)
+    KEY_FIELDS = ('backend_name',)
```

Comment:
@brittneyexline Can you review this change?  Looks like this was introduced in one of your recent PRs.  Thanks.

Reply:
I understand this change may have side-effects. However, I believe it is important to either (1) implement it so edX behaves in an expected way or (2) update the documentation (https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/tpa/tpa_integrate_open/tpa_oauth.html). The slug is mentioned nowhere in the documentation leading to users being unable to activate third party authentication.

Reply:
@brittneyexline Thoughts?

Reply:
@jaepetto you are right that we should update the documentation regarding how we are now using the slug field, I apologize that we haven't yet. However, as I said in the previous comment, we can't do this as it is without breaking functionality that edx.org is depending on. Maybe a quick way around this would be to override KEY_FIELDS for the OAuth2ProviderConfig for your needs - up to this point we have been mainly concerned about SAMLProviderConfig. I still believe the more proper approach would be to address the ticket I linked (https://openedx.atlassian.net/browse/ENT-957), but that is not as straightforward.

Reply:
@brittneyexline Sorry, can you clarify what you expectation of  @jaepetto is right now?